hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphTransaction.java

```diff
@@ -113,6 +115,8 @@ public GraphTransaction(HugeGraph graph, BackendStore store) {
         assert !this.indexTx.autoCommit();
 
         final HugeConfig conf = graph.configuration();
+        this.checkIdExist = conf.get(
+                            CoreOptions.VERTEX_CHECK_CUSTOMIZED_ID_EXIST);
```

Comment:
rename to:
this.checkVertexId = conf.get(CoreOptions.VERTEX_CHECK_CUSTOMIZED_ID)